server.js

```diff
@@ -1,24 +1,16 @@
 const ERR = require('async-stacktrace');
 const fs = require('fs');
-const path = require('path');
-const favicon = require('serve-favicon');
 const async = require('async');
-const express = require('express');
-const bodyParser = require('body-parser');
-const cookieParser = require('cookie-parser');
-const passport = require('passport');
 const http = require('http');
 const https = require('https');
 const blocked = require('blocked-at');
-const onFinished = require('on-finished');
-const uuidv4 = require('uuid/v4');
 const argv = require('yargs-parser') (process.argv.slice(2));
```

Comment:
While you are making changes to all the server stuff, could you also fix it to just use `yargs` properly rather than directly using the low-level `yargs-parser`? This has been bothering me :-) There is an example of `yargs` usage in `tools/load-test.js`.

Reply:
Sure thing. We currently have
```
// If there is only one argument, legacy it into the config option
```
Can we remove support for passing a config file name without the `--config` option?

Reply:
Yes, so long as the docker files aren't using it.

Reply:
`init.sh` seems to be using `--config`, so we should be good.

Reply:
👍